bokeh/core/json_encoder.py

```diff
@@ -243,7 +243,7 @@ def default(self, obj):
         elif isinstance(obj, collections.deque):
             return list(map(self.default, obj))
         elif isinstance(obj, Model):
-            return obj.ref
+            return obj.ptr
```

Comment:
This kind of change needs to occur under an optional feature flag that is off by default. We could imagine flipping it on for 2.0

Reply:
Surely Python API stability means that Python function calls continue to return the same results on the same inputs, over time. I would not be surprised if there are people who are testing their usage of Bokeh by validating JSON output.